### DISCUSSION TITLE
BLOB 支持的编码方式有 DICTIONARY ，但执行SQL 报错提示不支持

### DISCUSSION THREAD

**Discussion by wenmin864** [apache/iotdb#14313] [Q&A] [answered]
IoTDB 版本： 1.3.3
官网文档上写 BLOB 支持的编码方式有 DICTIONARY 
![企业微信截图_17333066756937](https://github.com/user-attachments/assets/f150d778-6c85-4c5a-a1c1-43e661743a84)

SQL 执行报错
![企业微信截图_17333092238500](https://github.com/user-attachments/assets/a1919480-b593-42ea-b5ce-606b1ffb631d)

**Comment by HTHou** [comment, marked as answer]
文档写的不太对，我们会尽快修改，感谢反馈。

**Reply by wenmin864** [reply]
OK
